content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -489,6 +489,11 @@ If a file is created and deleted, but has an open file descriptor,
 it continues to consume space. Quota tracking records that space accurately
 whereas directory scans overlook the storage used by deleted files.
 
+Quotas are to be enabled within user namespaces because user 
```

Comment:
I think this could be worded clearer:
```
To use quotas to track a pods resource usage, that pod must be in a user namespace. This is because the kernel restricts changes to projectIDs on the filesystem, which 
is essential for preserving the reliability of storage metrics 
calculated by quotas.
```